Test failure for tests/integration/cmor/_fixes/cmip5/test_bcc_csm1_1.py

I am noticing this test failure on multiple platforms including my laptop, Github Actions (see [GA](https://github.com/ESMValGroup/ESMValCore/runs/1651548703?check_suite_focus=true) output for both Python 3.8 and 3.9):
```
______________________________________________________ test_tos_fix_data _______________________________________________________
[gw1] linux -- Python 3.9.1 /home/valeriu/miniconda3/envs/esmvalcore39/bin/python3.9

    def test_tos_fix_data():
        """Test ``fix_data`` for ``tos``."""
        grid_lat = iris.coords.DimCoord(
            [20.0, 40.0],
            bounds=[[10.0, 30.0], [30.0, 50.0]],
            var_name='rlat',
            standard_name='grid_latitude',
        )
        grid_lon = iris.coords.DimCoord(
            [10.0, 20.0, 30.0],
            bounds=[[5.0, 15.0], [15.0, 25.0], [25.0, 35.0]],
            var_name='rlon',
            standard_name='grid_longitude',
        )
        latitude = iris.coords.AuxCoord(
            [[-40.0, -20.0, 0.0], [-20.0, 0.0, 20.0]],
            var_name='lat',
            standard_name='latitude',
            units='degrees_north',
        )
        longitude = iris.coords.AuxCoord(
            [[100.0, 140.0, 180.0], [80.0, 100.0, 120.0]],
            var_name='lon',
            standard_name='longitude',
            units='degrees_east',
        )
    
        # Create cube without bounds
        cube = iris.cube.Cube(
            np.full((2, 3), 300.0),
            var_name='tos',
            units='K',
            dim_coords_and_dims=[(grid_lat, 0), (grid_lon, 1)],
            aux_coords_and_dims=[(latitude, (0, 1)), (longitude, (0, 1))],
        )
        assert cube.coord('latitude').bounds is None
        assert cube.coord('longitude').bounds is None
    
        # Apply fix
        fix = Tos(None)
        fixed_cube = fix.fix_data(cube)
        assert fixed_cube is cube
        assert fixed_cube.coord('latitude').bounds is not None
        assert fixed_cube.coord('longitude').bounds is not None
        latitude_bounds = np.array([[[-40, -33.75, -23.75, -30.0],
                                     [-33.75, -6.25, 3.75, -23.75],
                                     [-6.25, -1.02418074021670e-14, 10.0, 3.75]],
                                    [[-30.0, -23.75, -13.75, -20.0],
                                     [-23.75, 3.75, 13.75, -13.75],
                                     [3.75, 10.0, 20.0, 13.75]]])
>       np.testing.assert_allclose(fixed_cube.coord('latitude').bounds,
                                   latitude_bounds)
E       AssertionError: 
E       Not equal to tolerance rtol=1e-07, atol=0
E       
E       Mismatched elements: 24 / 24 (100%)
E       Max absolute difference: 3.75
E       Max relative difference: 5.25762448e+13
E        x: array([[[-43.480762, -34.019238, -22.009619, -31.471143],
E               [-34.019238, -10.      ,   2.009619, -22.009619],
E               [-10.      ,  -0.538476,  11.471143,   2.009619]],...
E        y: array([[[-4.000000e+01, -3.375000e+01, -2.375000e+01, -3.000000e+01],
E               [-3.375000e+01, -6.250000e+00,  3.750000e+00, -2.375000e+01],
E               [-6.250000e+00, -1.024181e-14,  1.000000e+01,  3.750000e+00]],...

tests/integration/cmor/_fixes/cmip5/test_bcc_csm1_1.py:82: AssertionError
```
I can fix this asap by throttling the `rtol` value, but it looks to me as the differences are pretty darn big (O(1e1)) so a functional fix may be needed - any ideas @jvegasbsc ? :confused:

--- a/environment.yml
+++ b/environment.yml
@@ -11,3 +11,4 @@ dependencies:
   - graphviz
   - python>=3.6,<3.9  # if 3.7 lxml will not import correctly if <4.5.0
   - python-stratify
+  - scipy<1.6  # until ESMValGroup/ESMValCore/issues/927 gets resolved

--- a/package/meta.yaml
+++ b/package/meta.yaml
@@ -39,6 +39,7 @@ requirements:
     - graphviz
     - iris>=2.2.1
     - python-stratify
+    - scipy<1.6  # until ESMValGroup/ESMValCore/issues/927 gets resolved
     # Normally installed via pip:
     - cftime>=1.3.0  # years<999 get a 0 instead of empty space
     - cf-units

--- a/tests/integration/cmor/test_table.py
+++ b/tests/integration/cmor/test_table.py
@@ -364,7 +364,7 @@ class TestCustomInfo(unittest.TestCase):
         """Get none if a variable is not in the given table."""
         self.assertIsNone(self.variables_info.get_variable('Omon', 'badvar'))
 
-    def test_get_variable_tasConf5(self):
+    def test_get_variable_tasconf5(self):
         """Get tas variable."""
         CustomInfo()
         var = self.variables_info.get_variable('Amon', 'tasConf5')
@@ -373,7 +373,7 @@ class TestCustomInfo(unittest.TestCase):
                          'Near-Surface Air Temperature Uncertainty Range')
         self.assertEqual(var.units, 'K')
 
-    def test_get_variable_tasConf95(self):
+    def test_get_variable_tasconf95(self):
         """Get tas variable."""
         CustomInfo()
         var = self.variables_info.get_variable('Amon', 'tasConf95')

